src/gc/gc.cpp

```diff
@@ -26685,7 +26683,7 @@ void gc_heap::background_grow_c_mark_list()
         assert (new_c_mark_list);
         memcpy (new_c_mark_list, c_mark_list, c_mark_list_length*sizeof(uint8_t*));
         c_mark_list_length = c_mark_list_length*2;
-        delete c_mark_list;
+        delete [] c_mark_list;
```

Comment:
>delete [] [](start = 8, length = 9)

delete c_mark_list was intentional. there's no destructor to run for the elements of c_mark_list. we want to delete it as a single object.